build.sbt

```diff
@@ -1,13 +1,12 @@
-ThisBuild / scalaVersion := "2.13.13"
+ThisBuild / scalaVersion := "2.13.14"
 
 lazy val testSuite = (project in file("."))
   .disablePlugins(JUnitXmlReportPlugin) // Required to prevent https://github.com/scalatest/scalatest/issues/1427
   .settings(
     name := "advance-valuation-rulings-ui-tests",
     version := "0.1.0",
```

Comment:
our versions in ATs are all over the place, change this to 1.0.0 to align all of them, dont change anything in terms of build